dom/lists/DOMTokenList-iteration.html

```diff
@@ -5,46 +5,59 @@
 <script src=/resources/testharnessreport.js></script>
 <span class="   a  a b "></span>
 <script>
-  test(function() {
-    var list = document.querySelector("span").classList;
+  var list = document.querySelector("span").classList;
+  test(() => {
     assert_array_equals([...list], ["a", "b"]);
+  }, "classList");
 
-    var keys = list.keys();
-    assert_false(keys instanceof Array);
+  var keys = list.keys();
```

Comment:
I think it's better to have this inside the test function. In case it would throw. Same for `values()` and `entries()`